docs/0.4/glossary/glossary.md

```diff
@@ -6,79 +6,143 @@
   https://creativecommons.org/licenses/by/4.0/
 -->
 
-## Biome
+<h3 class="glossary-header" id="biome">
+Biome
+</h3>
 
-Biome is a module in libsplinter containing several submodules that provide
-support for user management, user credential management, and private key
-management.
+<p class="glossary-definition">
+<b class="glossary-entry">Biome</b> is a module in libsplinter containing
```

Comment:
I'm not sure making this `glossary-entry` fuchsia or bold is necessary; IMO it's visually distracting, and the word is already in big fuchsia letters in the H3.

Reply:
I was basing this off of glossary examples that we had talked about, mainly 
https://pulsar.apache.org/docs/en/reference-terminology/, but if you think that it's distracting, I am not tied to this design.